android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/LanguageSetting.kt

```diff
@@ -42,4 +42,84 @@ enum class LanguageSetting(private val languageSetting: String) {
                 throw IllegalArgumentException("The language setting $languageSetting is not mapped.")
         }
     }
+
```

Comment:
This is here right now just for running early CI tests. (Had issues with the middleware test in the past and would like to see if it is resolved.) See dependency where it is added in #5325.